openapi/code/package.go

```diff
@@ -325,15 +352,56 @@ func (pkg *Package) HasWaits() bool {
 	return false
 }
 
+func (pkg *Package) getTagByServiceName(name string, spec *openapi.Specification) (*openapi.Tag, error) {
+	for _, tag := range spec.Tags {
+		if tag.Service == name {
+			return &tag, nil
+		}
+	}
+	return nil, fmt.Errorf("tag %s not found", name)
+}
```

Comment:
Should this be a method on `openapi.Specification`? Doesn't seem to have anything to do with package.